dftimewolf/lib/collectors/gce_disk_copy.py

```diff
@@ -62,8 +62,8 @@ def SetUp(self,
             destination_project_name: Optional[str],
             source_project_name: str,
             destination_zone: str,
-            remote_instance_names: Optional[str | List[str]],
-            disk_names: Optional[str | List[str]],
+            remote_instance_names: Union[str, List[str]],
+            disk_names: Union[str, List[str]],
```

Comment:
Can also be `None`, so add it to the union (that's what `Optional` means)

Reply:
Good point. I updated the code.